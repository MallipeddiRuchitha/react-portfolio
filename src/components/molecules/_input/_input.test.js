import React from 'react';
import { shallow ,mount} from 'enzyme';
import Input from './_input';
const mockOnChange = jest.fn();
const value="";
const label="name";
describe("Input component",() =>{
    test("Matches the snapshot",() => {
        const input = shallow(<Input onChange={mockOnChange} label={label} value={value}/>);
        expect(input).toMatchSnapshot();

    })
    test("input element is rendered",() => {
        const input = shallow(<Input onChange={mockOnChange} label={label} value={value}/>);
        expect(input.find("#input").exists()).toEqual(true);
        
    });
   
   
   

});