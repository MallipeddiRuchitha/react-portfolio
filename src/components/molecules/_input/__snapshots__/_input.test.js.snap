// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Input component Matches the snapshot 1`] = `
<div
  id="input"
>
  <WithStyles(ForwardRef(Typography))
    variant="h5"
  >
    name
  </WithStyles(ForwardRef(Typography))>
  <InputField
    onChange={[MockFunction]}
    test="name"
    value=""
  />
</div>
`;
